database/migrations/2017_01_16_092942_add_tasks_priority.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace {
+
+    use Illuminate\Database\Migrations\Migration;
+    use App\GenericModel;
+
+    class AddTasksPriority extends Migration
+    {
+        /**
+         * Run the migrations.
+         *
+         * @return void
+         */
+        public function up()
+        {
+            GenericModel::setCollection('tasks');
+            $tasks = GenericModel::all();
+            foreach ($tasks as $task) {
+                $task->update([
```

Comment:
Run update only if priority is not set - beause this migration will run multiple times we don't want it to overwrite data each time it runs